core/domain/rights_manager.py

```diff
@@ -166,6 +184,17 @@ def is_editor(self, user_id):
         """
         return bool(user_id in self.editor_ids)
 
+    def is_translator(self, user_id):
+        """Checks whether given user is translator of activity.
+
+        Args:
+            user_id: str or None. Id of the user.
+
+        Returns:
+            bool. Whether user is in activity translator.
```

Comment:
in --> an

Reply:
Done! (similar changes in above docstrings)